src/components/Tabs/Tabs.scss

```
@@ -0,0 +1,12 @@
+.button {
+  padding: 10px;
+  width: 100%;
+}
+
+.button:hover {
+  background-color: rgb(250, 156, 156);
```

Можна не лише на **hover**

---

А, і в тебе підключений **scss**

```suggestion
.button {
  padding: 10px;
  width: 100%;

  &:hover {
    background-color: rgb(250, 156, 156);
    }
  }
```